### PR TITLE
PW-1447: Fixes for Dankort, elo, hipercard, reformat appinfo, add checkout com…

### PR DESCRIPTION
**Pull request by Aleffio**
…ponent version to css

### CODE REVIEW

File: app/code/community/Adyen/Payment/Model/Api.php [hunk 1]
```diff
@@ -125,8 +125,7 @@ public function authorisePayment($payment, $amount, $paymentMethod)
         if (!Mage::app()->getStore()->isAdmin() && Mage::getStoreConfigFlag('payment/adyen_cc/enable_threeds2', $storeId)) {
             $request = $this->setThreeds2Data($request, $payment);
         }
-        $request['applicationInfo'] = array(new Adyen_Payment_Model_Adyen_Data_ApplicationInfo());
-
+        $request = $this->setApplicationInfo($request);
```
Comment (rikterbeek) [inline app/code/community/Adyen/Payment/Model/Api.php]: I believe we can remove the Adyen_Payment_Model_Adyen_Data_ApplicationInfo or not probably still being used by modifications ?

File: app/code/community/Adyen/Payment/Model/ProcessNotification.php [hunk 2]
```diff
@@ -1263,6 +1263,9 @@ protected function _manualCaptureAllowed()
             case 'laser':
             case 'paypal':
             case 'sepadirectdebit':
+            case 'dankort':
+            case 'elo':
+            case 'hipercard':
```
Comment (rikterbeek) [inline app/code/community/Adyen/Payment/Model/ProcessNotification.php]: are we missing this in M2 as well ?

File: app/code/community/Adyen/Payment/Model/Api.php (hunk 1 above)
Comment (Aleffio) [reply app/code/community/Adyen/Payment/Model/Api.php]: indeed it is used in modifications

File: app/code/community/Adyen/Payment/Model/ProcessNotification.php (hunk 2 above)
Comment (Aleffio) [reply app/code/community/Adyen/Payment/Model/ProcessNotification.php]: will check and create a ticket if needed

**Pull request merged**